src/test/ui/rfc-2086-irrefutable_let_patterns/deny-irrefutable-if-let.stderr

```diff
@@ -0,0 +1,16 @@
+error: irrefutable if-let pattern
+  --> $DIR/deny-irrefutable-if-let.rs:13:5
+   |
+LL | /     if let _ = 5 {
```

Comment:
Hmm... I would expect the lint to underline and point towards `_` as the source so that it is actionable for the user...

cc @estebank

Reply:
That was accomplished before in these lines

https://github.com/rust-lang/rust/pull/55639/files#diff-a2cadcdf8ac2f01fa5a02f2b4e0a8267L386

https://github.com/rust-lang/rust/pull/55639/files#diff-a2cadcdf8ac2f01fa5a02f2b4e0a8267L417

We just need to keep that logic around with a fallback of the entire block's span.